doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1125,6 +1165,28 @@ Here are the syntax elements used by the various notation commands.
              due to legacy notation in the Coq standard library.
              It can be turned on with the ``-w disj-pattern-notation`` flag.
 
+.. note::
+
+   As of version 8.13, the entry ``ident`` is a deprecated alias for
+   ``name``. In the future, it is planned to strictly parse an
```

Comment:
```suggestion
   ``name``. In the future, we plan to strictly parse an
```

Reply:
In general, I don't like the `we` so much as it would require to define who is "we" (which I would not be able to do).

Reply:
It would mean the doc authors/Coq team.  In practice, I think you would make the change (and we, the rest of team, support that).  French doesn't use "we" this way?

We (there it is again!) try to avoid using the passive voice ("... it is planned") where possible.  The sentence is less readable with an unspecified subject, although in reference manuals it's often unavoidable.

Reply:
>  French doesn't use "we" this way?

French can use "we" (i.e. "nous") this way. The passive form would be fine here in French ("Il est prévu à terme que cette entrée corresponde strictement aux identifiers, à l'exclusion de _" or something like that).

In any case, I find the passive form a more informal promise than a "we" which would correspond to a "we" at the date of today and might not be the exact same "we" in one year from now on.